### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by IsabelleTornqvist**

### CODE REVIEW

**Review by ghost — DISMISSED** [review]
so fetch
recommended: 
How to run the server instead of HOW TO RUN SERVER
IP instead of ip

File: README.md [hunk 1]
```diff
@@ -11,3 +11,8 @@ Tim Jonasson, Pontus Laestadius, Rashad Kamsheh, Sebastian Fransson, Kosara Gole
 
 - [gson library](https://github.com/google/gson)
 is a Java serialization/deserialization library to convert Java Objects into JSON and back
+
+# HOW TO RUN SERVER
+1. Edit ip in usercoordinator.erl: change to your local ip
```
Comment (Sebastian-Fransson) [inline README.md]: should be diagramcoordinator

**Review by Sebastian-Fransson — DISMISSED** [review]

**Review by kosara-g — DISMISSED** [review]
Awesome!

**Review by Sebastian-Fransson — APPROVED** [review]

**Review by ghost — APPROVED** [review]
pull request of the year

**Pull request merged**